gravitee-apim-repository/gravitee-apim-repository-jdbc/src/main/resources/liquibase/changelogs/v4_5_0/06_add_scoring_table.yml

```diff
@@ -9,6 +9,7 @@ databaseChangeLog:
               - column: { name: id, type: bigint, autoIncrement: true, constraints: { primaryKey: true, nullable: false } }
```

Comment:
Changing this file might cause some start up issues because of a failing checksum validation during liquibase execution.